src/session/client-certificate.c

```diff
@@ -56,7 +56,7 @@ read_proc_self_cgroup (size_t *out_length)
   if (fp == NULL)
     {
       warn ("Failed to open /proc/self/cgroup");
```

Comment:
I guess you want to keep the copy for the journal?

Reply:
Yes, as we don't see the protocol messages in the journal on failures, so the warn()/fail() are for tests and admins.